cmd/lvh/runner/runner.go

```diff
@@ -0,0 +1,149 @@
+package runner
+
+import (
+	"fmt"
+	"os/exec"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"golang.org/x/sys/unix"
+)
+
+var (
+	rcnf RunConf
+
+	mount string
+	ports []string
+)
+
+func RunCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:          "run",
+		Short:        "run/start VMs based on generated base images and kernels",
+		SilenceUsage: true,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			var err error
+
+			rcnf.Logger = logrus.New()
+
+			rcnf.HostMount = mount
+
+			rcnf.ForwardedPorts, err = parsePorts(ports)
+			if err != nil {
+				return fmt.Errorf("Port flags: %w", err)
+			}
+
+			t0 := time.Now()
+
+			err = StartQemu(rcnf)
+			dur := time.Since(t0).Round(time.Millisecond)
+			fmt.Printf("Execution took %v\n", dur)
+			if err != nil {
+				return fmt.Errorf("Qemu exited with an error: %w", err)
+			}
+
+			return nil
+		},
+	}
+
+	cmd.Flags().StringVar(&rcnf.Image, "image", "", "VM image file path")
+	cmd.MarkFlagRequired("image")
+	cmd.Flags().StringVar(&rcnf.KernelFname, "kernel", "", "kernel filename to boot with. (if empty no -kernel option will be passed to qemu)")
+	cmd.Flags().BoolVar(&rcnf.QemuPrint, "qemu-cmd-print", false, "Do not run the qemu command, just print it")
+	cmd.Flags().BoolVar(&rcnf.DisableKVM, "qemu-disable-kvm", false, "Do not use KVM acceleration, even if /dev/kvm exists")
+	cmd.Flags().BoolVar(&rcnf.Daemonize, "daemonize", false, "daemonize QEMU after initializing")
+	cmd.Flags().StringVar(&mount, "mount", "", "Mount a directory")
```

Comment:
```suggestion
	cmd.Flags().StringVar(&rcnf.HostMount "host-mount", "", "Mount the specified host directory in the VM using a 'host_mount' tag")
```

Reply:
I'd prefer having a way to specify the directory and the tag from the command line, but better documenting the switch is OK with me at this point.